tests/handlers/test_agent.py

```diff
@@ -1,3 +1,14 @@
+import uuid
+
+import pytest
+from fastapi import WebSocket
+
+from apollo import app
+from apollo.lib.websocket_manager import WebSocketManager
+from apollo.handlers.agent import close_websocket_connection
```

Comment:
import ordering

Reply:
Also odd to see this import, handler tests should be integrated through a test client.